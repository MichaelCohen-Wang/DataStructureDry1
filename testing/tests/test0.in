get_treasure 455292
add_ship 50666 24
add_ship 501530 24
remove_ship 50666
add_ship 312740 4
add_pirate 491061 312740 16
remove_ship 501530
get_treasure 491061