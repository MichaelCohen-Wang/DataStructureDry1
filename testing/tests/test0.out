get_treasure: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
remove_ship: SUCCESS
get_treasure: SUCCESS, 16
